### PR TITLE
Hello World example running

### PR DESCRIPTION
**Pull request by lucasvfventura**
Remove singleton on the system messages messages
Create simple ThreadDispatcher
Create PoisonPill message
Fix some properties
Create runnable hello world example

### CODE REVIEW

**Comment by alexpantyukhin** [comment]
Why there is need to remove singleton?

**Comment by lucasvfventura** [comment]
We use a lot of times the function isinstance witch fails using the @utils.singleton. Do you know a workaround this?

**Comment by alexpantyukhin** [comment]
the problem that tests fail now is that you use `assert_called_once`. Try `assert_called_once_with()`

**Comment by lucasvfventura** [comment]
I fixed it xD

File: examples/hello_world.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        message = context.message
+        if isinstance(message, HelloMessage):
+            print(message.text)
+
+
+if __name__ == "__main__":
+    props = actor.from_producer(lambda: HelloActor())
+    pid = actor.spawn(props)
+    pid.tell(HelloMessage('Hello World!'))
+    input()
```
Comment (ipinak) [inline examples/hello_world.py]: Do you need this?

Comment (lucasvfventura) [inline examples/hello_world.py]: This is the example... All implementations has it

File: protoactor/dispatcher.py [hunk 2]
```diff
@@ -14,18 +15,26 @@ def throughput(self) -> int:
...
-            async_loop = asyncio.get_event_loop()
-            task = asyncio.wait(async_loop.create_task(runner()))
-            async_loop.run_until_complete(task)
-            async_loop.close()
-
-        p = Process(target=run_async, args=runner)
+        p = Process(target=_run_async, args=(runner,))
         p.start()
+
+class ThreadDispatcher(AbstractDispatcher):
```
Comment (ipinak) [inline protoactor/dispatcher.py]: Do we need the 2nd dispatcher?

Comment (lucasvfventura) [inline protoactor/dispatcher.py]: The idea is to offer different dispatchers due Python nature one might me more appropriate in certain cases than others

**Pull request merged**